apps/demo-lit-connect-modal-next/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "demo-lit-connect-modal-next",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@ethersproject/providers": "^5.7.2",
+    "@lit-protocol/auth-browser": "file:../../packages/auth-browser",
+    "@lit-protocol/constants": "file:../../packages/constants",
```

Comment:
Should we use publish versions instead of local links? incase we move it to the `apps` repo?